Updated handover process in onboarding documentation (#13688)
PostHog/posthog.com: contents/handbook/onboarding/sales-handover.md

showTitle: true
---
### Initial qualification

#### Direct handover - skipping onboarding

> If you see that a customer is spending **more than ~$1,500 monthly**, evaluate whether their usage looks stable and legitimate, and make sure that MRR doesn't come from an unwanted event spike or misconfiguration issue.

Reviewer: ```suggestion
If you see that a customer is spending **more than $1,000 monthly**, evaluate whether their usage looks stable and legitimate, and make sure that MRR doesn't come from an unwanted event spike or misconfiguration issue.
```

If that's the case, you can pass the account to Sales even **without speaking with the customer first**, as long as you’ve confirmed that the high spend is intentional. The goal is to react quickly to healthy, high-spend accounts—but avoid passing through problematic ones.

Be courteous and always provide context on the account (e.g., what you've noticed in Metabase, etc.) to the Sales person assigned to your lead.

#### Handover during onboarding

> While talking with customers or analyzing the account, conduct a discovery to understand the reason behind their high spend and assess whether there's potential for stable spend or usage moving forward. If they’re happy continuing with PostHog, you can mention our discounted pre-paid plan, which helps them save ~20%. However, if they prefer paying monthly, they are more than welcome to do so!

Reviewer: ```suggestion
While talking with customers or analyzing the account, do some discovery to understand the reason behind their high spend and assess whether there's potential for stable spend or usage moving forward. If they’re happy continuing with PostHog, you can mention our discounted pre-paid plan, which helps them save ~20%. However, if they prefer paying monthly, they are more than welcome to do so!
```
